src/packages.builds

```diff
@@ -13,7 +13,7 @@
 
     <!-- Use the same package version for all output packages. -->
     <DoNotGeneratePackageVersion>true</DoNotGeneratePackageVersion>
-    <PackageVersion>2.1.0-preview2-$(BuildNumberMajor)-$(BuildNumberMinor)</PackageVersion>
+    <PackageVersion>2.1.0-preview3-$(BuildNumberMajor)-$(BuildNumberMinor)</PackageVersion>
```

Comment:
Would be nice to wait to merge `master` into `release/2.1` again until we start doing preview3 so this change works naturally. For now I guess this has to be reverted.

Reply:
We plan to do a full branch merge in the core repo's on Friday but still for preview2 so I will revert this one change.